packages/docs/next.config.js

```diff
@@ -2,6 +2,7 @@ const withNextra = require("nextra")({
   theme: "nextra-theme-docs",
   themeConfig: "./theme.config.tsx",
   latex: true,
+  defaultShowCopyCode: true,
```

Comment:
I think we should do it in a separate PR.

Reply:
Meh, I agree but it's fineee

Reply:
Actually I changed my mind, let's remove it for now and open a new PR

Reply:
Sorry about that! Thanks for fixing that @bengtlofgren